packages/core/src/routes/interaction/middleware/koa-session-sign-in-experience-guard.ts

```diff
@@ -20,13 +20,15 @@ export default function koaSessionSignInExperienceGuard<
     const interaction = await provider.interactionDetails(ctx.req, ctx.res);
     const { event, identifier, profile } = ctx.interactionPayload;
 
+    if (event === 'forgot-password') {
```

Comment:
as this is `koaSessionSignInExpecrienceGuard`, should we also consider 'register' event as not applicable?

Reply:
nope we guard the register methods as well.